cpp_implementation/simplerng.cc

```diff
@@ -0,0 +1,28 @@
+#include "simplerng.h"
+
+uint32_t SimpleRNG::SimpleRNG_rand(uint32_t &rng_state)
```

Comment:
Because they are now mainspaced, `SimpleRNG::rand` is good enough for a qualified name, right? :-)

Reply:
Ah, of course 🙃